mobile/src/store/store.ts

```diff
@@ -19,6 +20,7 @@ const extraArgument = {
   storage,
   usersApi,
   permissionsApi,
+  coursesApi,
```

Comment:
You have already added it at the top

Reply:
But at the top, we have some imports, at the bottom -  extraArgument
![image](https://user-images.githubusercontent.com/82529236/185571771-720eedf5-be8f-4c45-9ba4-7cb3c8f51102.png)

Reply:
You are right, didn't notice it